DataStructures/tree.go

```diff
@@ -0,0 +1,100 @@
+// Go's concurrency primitives make it easy to
```

Comment:
do you know what this means or did you copy paste this 😆

Reply:
only be merged,if you explain what this line means 😸

Reply:
@aswinzz @Anupam-Dagar

Reply:
Concurrency is the composition of independently executing computations.

Reply:
will give you 50 points,if you write a 200 words + blog post on concurrency in go with images,example etc. 

and put on medium or somewhere 😄 
**only for you**